elastic/kibana
# [Actions][ServiceNow] Allow to close serviceNow incident when alert resolves

## Summary

Fixes: https://github.com/elastic/kibana/issues/170522

This PR allows to `close service now incident` when alert is `recovered`

SN connector form shows only `correlation_id` field as it is mandatory field to close an incident. 

![Screenshot 2023-11-27 at 11 52 36](https://github.com/elastic/kibana/assets/117571355/1d722153-f77a-484a-b17b-13489f9d7666)

**How to test:**
- Create a rule and select serviceNow ITSM action with Run when option as Recovered
- Verify that it closes an incident in SN when Alert is recovered


### Checklist

Delete any items that are not applicable to this PR.


### For maintainers


## Release notes
Auto close ServiceNow incidents when alerts are resolved
